docs/commandline.rst

```diff
@@ -139,3 +139,19 @@ For example:
 ::
 
     kinto rebuild-quotas --ini=config/postgresql.ini
+
+Flush Cache
+-----------
+
+Clears the Memory Backend Cache This can be useful for
```

Comment:
The *cache backend*, nothing more. Since it does not have to be memory, for example if memcached is configured...

Also add a `.` before `This can`....

Reply:
Done